packages/prettier-config/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@open-wc/prettier-config",
+  "version": "0.1.0",
```

Comment:
this will release 0.2.0 within the ci as standard version will bump minor for 'feat' commits...
probably best to set it to 0.0.0 and let the ci release 0.1.0
=> again not blocking we can merge as is if you want